### PR TITLE
Wrist Slot (Wear watches with gloves!)

### PR DESCRIPTION
**Pull request by alsoandanswer**
title.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_wrists.dm [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/datum/gear/wrists
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_wrists.dm:1]: Bracelets should be added to this section too.

**Review by skull132 — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]
This better not break anything.

File: code/_onclick/hud/hud.dm [hunk 2]
```diff
@@ -187,43 +187,47 @@ datum/hud/New(mob/owner)
...
 				switch(hud_data["slot"])
 					if(slot_head)
-						if(H.head)      H.head.screen_loc =      hud_data["loc"]
+						if(H.head)		H.head.screen_loc =		hud_data["loc"]
 					if(slot_shoes)
-						if(H.shoes)     H.shoes.screen_loc =     hud_data["loc"]
+						if(H.shoes)		H.shoes.screen_loc =	hud_data["loc"]
 					if(slot_l_ear)
-						if(H.l_ear)     H.l_ear.screen_loc =     hud_data["loc"]
+						if(H.l_ear)		H.l_ear.screen_loc =	hud_data["loc"]
```
Comment (NonQueueingMatt) [inline code/_onclick/hud/hud.dm:194]: Indent all of these properly below the ifs if you're changing this.

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 3]
```diff
@@ -1579,6 +1632,16 @@ There are several things that need to be remembered:
...
 
+/mob/living/carbon/human/proc/check_draw_wrists()
+	if (!wrists)
+		return 0
+	else if (wrists.flags_inv & ALWAYSDRAW)
+		return 1
+	else if (wrists && (wrists.flags_inv & HIDEWRISTS))
+		return 0
+	else
+		return 1
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/update_icons.dm:1643]: true/false

File: code/modules/mob/mob_helpers.dm [hunk 4]
```diff
@@ -1187,6 +1189,6 @@ proc/is_blind(A)
 
 /mob/proc/remove_deaf()
 	sdisabilities &= ~DEAF
-
+
```
Comment (NonQueueingMatt) [inline code/modules/mob/mob_helpers.dm:1192]:
```suggestion

```

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Pull request merged**